src/test/java/fi/metatavu/kuntaapi/test/utils/PtvDownload.java

```diff
@@ -83,23 +266,49 @@ private ServiceChannelType resolveChannelType(String type) {
         return null;
     }
   }
+  
+  private String printOrganizationServicesList(int index, File organizationFile) throws IOException {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "index". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)